src/reducers/history.js

```diff
@@ -134,15 +145,18 @@ export default function reduce(state = initialState, action) {
         ...state,
         messages: [...state.messages, { ...payload, state: 'pending' }],
         showNoContent: false,
+        receivedMessageViaSocket: false,
       }
     case types.ADD_NEW_MESSAGE: {
       if (payload.channelId !== state.channel.id) return state
       const scrollTo = payload.author.id === state.user.id ? payload.id : null
       return {
         ...state,
         scrollTo,
+        scrollToAlignment: null,
```

Comment:
What's the purpose to reset this?

Reply:
Avoiding inconsistent states that lead to unexpected behaviour. Does that make sense?